index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio Homepage</title>
+</head>
+
+<body>
+  <div id="toolbar">
+    <div id="logoPlaceholder">
+      <p id="logoText">Welcome</p>
+    </div>
+    <input id="headerButton" type="image" src="Images/Enabled.png" alt="Menu">
+  </div>
+  <div id="headline">
+    <div class="primaryText" id="primaryText">
+      <p class="primaryText" id="primaryName">
+        Hey there. I'm Ricardo
+      </p>
+      <p class="primaryText" id="primaryJob">
+        I'm a software developer
+      </p>
+    </div>
+    <p id="supportingText">
+      I can help you build a product, feature or website.
+      Look through some of my work and experience!
+      If you like what you see and have a project you need coded,
+      don’t hestiate to contact me.
+    </p>
+    <div id="socialMediaIcons">
+      <input class="socialMediaIcons" type="image" id="twitterIcon" src="Images/twitter.png" alt="Twitter">
+      <input class="socialMediaIcons" type="image" id="linkedinIcon" src="Images/Linkedin.png" alt="Linkedin">
+      <input class="socialMediaIcons" type="image" id="mediumIcon" src="Images/medium.png" alt="Medium">
+      <input class="socialMediaIcons" type="image" id="githubIcon" src="Images/github.png" alt="GitHub">
+      <input class="socialMediaIcons" type="image" id="angelistIcon" src="Images/angellist.png" alt="Angelist">
+    </div>

```

Comment:
Since this is a list, kindly make use of the `ul > li` tags to list all your icons. You might need to use the `list-style: none` on the `ul` CSS selector to remove the bullet points and perhaps `padding: 0` if you wish to remove the indentation of the `li` elements.

- Also the social media icons should use the `a` tags since they are links that redirect users to the relevant social media website.